docs/es/docs/alternatives.md

```diff
@@ -0,0 +1,424 @@
+
+
+## Herramientas anteriores
+
+### <a href="https://www.djangoproject.com/" class="external-link" target="_blank">Django</a>
+
+Es el frameworks más popular de Python y es ampliamente conflable. Es utilizado para construir sistemas como Instagram.
+
+Está  muy  acoplado con bases de datos relacionales (como MySQL o PostgreSQL), por lo que no es fácil tener una base de datos NoSQL (como Couchbase, MongoDB, Cassandra, etc.) como motor principal de almacenamiento.
+
+Fue creado para generar el HTML del backend, no para crear API utilizadas por interfaces modernas (como React, Vue.js y Angular) o para que otros sistemas (como dispositivos de <abbr title = "Internet of Things"> IoT </ abbr > ) se comuniquen con él.
+
+### <a href="https://www.django-rest-framework.org/" class="external-link" target="_blank">Django REST Framework</a>
+
+Django REST framework fue creado para ser un kit flexible  de herramientas  para construir APIs web usando Django por debajo, y así mejorar sus capacidades API.
+
+Es utilizado por muchas compañías, como Mozilla, Red Hat y Eventbrite.
+
+Fue uno de los primeros ejemplos de **documentación automática de API**, siendo ésta, en particular, unas de las primeras ideas que inspiraron  "la búsqueda" de **FastAPI**.
+
+
+!!! nota
+    Django REST Framework fue creado por Tom Christie. El mismo creador de Starlette y Uvicorn, sobre los que **FastAPI** se apoya.
+
+
+!!! mirar "Inspiraron a **FastAPI** en"
+   Tener una interfaz web con la documentación automática de la API.
+
+### <a href="http://flask.pocoo.org/" class="external-link" target="_blank">Flask</a>
+
+Flask es un "microframework" que no incluye integraciones con bases de datos ni muchas cosas que vienen por defecto en Django.
+
+Esta simplicidad y flexibilidad nos permiten hacer cosas como utilizar bases de datos NoSQL como sistema de almacenamiento  principal de datos.
+
+Por ser tan  simple, es relativamente intuitivo de aprender, aunque la documentación se vuelve algo técnica en algunos apartes.
+
+También se utiliza comúnmente para otras aplicaciones que no necesariamente requieran  bases de datos, gestión de usuarios, o cualquiera de las tantas características que vienen por defecto en Django. Aunque, vale aclarar, muchas de estas características se pueden agregar con complementos.
+
+Este desacoplamiento de sus partes, y el hecho de ser un "microframework" que se puede  extender para cubrir exactamente lo que se requiere, era una característica clave que yo quería conservar.
+
+Dada la simplicidad de Flask, parecía una buena combinación para construir API. Ahora tenía que hallar un "Django REST Framework" para Flask.
+
+Given the simplicity of Flask, it seemed like a good match for building APIs. The next thing to find was a "Django REST Framework" for Flask.
```

Comment:
Hi @jorgerpo, thank you for your super fast reply!  🚀 

As you can check here, from this line forward this file is in English.